libottdadmin2/packets/__init__.py

```diff
@@ -6,9 +6,11 @@
 
 from libottdadmin2.packets.base import Packet
 from libottdadmin2.packets.admin import (
+    AdminAuthResponse,
     AdminChat,
     AdminGamescript,
     AdminJoin,
+    AdminJoinNew,
```

Comment:
Should this not be something like `AdminJoinSecure`, since this only applies to secure connections?

Reply:
Good point. It was already renamed in OpenTTD, but I never fixed that here.